packages/angular/cli/models/command.ts

```diff
@@ -51,47 +55,76 @@ export abstract class Command<T = any> {
     return true;
   }
 
-  printHelp(_options: T): void {
-    this.printHelpUsage(this.name, this.arguments, this.options);
+  printHelp(commandName: string, description: string, options: any): void {
+    if (description) {
+      this.logger.info(description);
+    }
+    this.printHelpUsage(commandName, this.options);
     this.printHelpOptions(this.options);
   }
 
-  protected printHelpUsage(name: string, args: string[], options: Option[]) {
+  private getArgIndex(def: OptionSmartDefault | undefined): number {
+    if (def === undefined || def.index === undefined || typeof def.index !== 'number') {
+      return 99999;
```

Comment:
I don't think we'll get to 99999 args but this looks like it should be a non-number type, probably `null`.